releases/release-1.23/README.md

```diff
@@ -37,51 +37,52 @@ description: |
+- **Tuesday, November 30, 2021**: Week 15 - Docs must be completed and reviewed
+- **Tuesday, December 7, 2021**: Week 16 - Kubernetes v1.23.0 released
 - **TBD**: Week 17 - Release Retrospective
 
 ## Timeline
 
 | **What** | **Who** | **When** | **WEEK** | **CI SIGNAL** |
 |---|---|-------|---|---|
-| Start of Release Cycle | Lead | Mon August 23rd 2021 | week 1 | [master-blocking] |
-| Start Enhancements Tracking | Enhancements Lead | Mon August 23rd 2021| Week 1 | |
-| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25 2021 | Week 1 | |
-| Schedule finalized | Lead | Thur August 26 2021 | week 1 | |
-| Team finalized | Lead | Fri August 27 2021 | week 1 | |
-| **Begin [Enhancements Freeze]** (23:59 PDT) | Enhancements Lead | TBD | TBD | [master-blocking], [master-informing] |
+| Start of Release Cycle | Lead | Mon August 23, 2021 | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Mon August 23, 2021| week 1 | |
+| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25, 2021 | week 1 | |
+| Schedule finalized | Lead | Thu August 26, 2021 | week 1 | |
+| Team finalized | Lead | Fri August 27, 2021 | week 1 | |
+| Start Release Notes Draft | Release Notes Lead | Tue September 7, 2021 | week 3 | |
+| **Begin [Enhancements Freeze]** (23:59 PDT) | Enhancements Lead | Thu September 9, 2021 | week 3 | [master-blocking], [master-informing] |
 | 1.23.0-alpha.2 released | Branch Manager ([@TBD](TBD)) | TBD | TBD | |
 | 1.23.0-alpha.3 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| KubeCon NA + Co-located events | | Oct 11-15 2021 | | Week 8 |
+| KubeCon NA + Co-located events | | October 11-15, 2021 | week 8 | |
+| 1.23.0-alpha.4 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
+| **Begin [Burndown]** (MWF meetings) | Lead | Mon November 1, 2021 | week 11 | [1.23-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | Mon November 1, 2021 | week 11 | |
+| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon November 1, 2021 | week 11 | |
+| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Tue November 2, 2021 | week 11 | |
+| Burndown Meetings daily| Lead | Mon November 15, 2021 | week 13 | |
+| **Begin [Code Freeze]** (18:00 PST) | Branch Manager | Tue November 16, 2021 | week 13 | |
 | 1.23.0-beta.0 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
-| **Begin [Burndown]** (MWF meetings) | Lead | TBD | TBD | [1.23-blocking], [master-blocking], [master-informing] |
-| **Call for [Exceptions][Exception]** | Lead | TBD | TBD | |
-| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | TBD | TBD | |
-| **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | TBD | TBD | |
+| Docs deadline - Open placeholder PRs | Docs Lead | Thu November 18, 2021 | week 13 | |
 | 1.23.0-beta.1 released | Branch Manager ([TBD](TBD)) | TBD | TBD | |
```

Comment:
@puerco we discussed that today, do you think we can drop beta.1?

Reply:
@reylejano There were folks who requested a beta.1 in recent releases. In theory we would not have much time within 1 week between beta.0 and rc.0, so let's drop that line for now.